File: Java/FizzBuzz.java
// function to print fizz for multiples of 3 and buzz in multiples of 5 and fizzbuzz for multiples of both 3 and 5
// Author: @Rushil9

import java.util.List;
import java.util.ArrayList;

public class FizzBuzz {
    public static List<String> fizzBuzz(int n) {
        List<String> lt = new ArrayList<>();
        String str = "";
        for (int i = 1; i <= n; i++) {
            if (i % 3 == 0 && i % 5 != 0) {
                lt.add("Fizz");
            }
            if (i % 5 == 0 && i % 3 != 0) {
                lt.add("Buzz");
            }
            if (i % 3 == 0 && i % 5 == 0 && i != 1) {
                lt.add("FizzBuzz");
            }
            if (i % 3 != 0 && i % 5 != 0) {
                lt.add(Integer.toString(i));
            }

        }
        return lt;
    }

    public static void main(String[] args) {
        String out = String.join("\n", fizzBuzz(100));
        System.out.println(out);
    }
}

File: Elixir/FizzBuzz.ex
defmodule FizzBuzz do
  def fizz do
    (1..100)
    |> Enum.map(&buzz/1)
    |> Enum.each(&IO.puts/1)
  end

  defp buzz(i) when rem(i, 15) == 0, do: "FizzBuzz"
  defp buzz(i) when rem(i, 5)  == 0, do: "Buzz"
  defp buzz(i) when rem(i, 3)  == 0, do: "Fizz"
  defp buzz(i),                      do: Integer.to_string(i)
end